annotation-interface/pages/bricks/index.tsx

```diff
@@ -0,0 +1,110 @@
+                                    </a>
+                                </Link>
+                            </div>
+                            {(categoryIndex > 0
+                                ? bricks.filter(
+                                      (brick) => brick.usage === category,
+                                  )
+                                : bricks
+                            ).map((brick, index) => (
+                                <p
+                                    key={index}
+                                    className="border-t border-black border-opacity-10 py-3 pl-3"
+                                >
+                                    <Link
+                                        key={index}
+                                        href={`/bricks/${brick._id}`}
+                                    >
+                                        <a className="self-start mr-2">
+                                            {brick.translations.find(
+                                                (translation) =>
+                                                    translation.language ===
+                                                    display.language,
+                                            )?.text ?? (
+                                                <span className="inline-flex align-top">
+                                                    <ExclamationIcon className="h-5 w-5 mr-2 pt-1" />
+                                                    This Brick is not translated
+                                                    to {display.language}.
+                                                </span>
+                                            )}
+                                        </a>
+                                    </Link>
+                                    <button
+                                        className="border border-black border-opacity-20 text-xs px-2 rounded-full whitespace-nowrap font-semibold align-text-top mr-2"
+                                        onClick={() =>
+                                            display.setCategoryIndex(
+                                                displayCategories.indexOf(
+                                                    brick.usage,
+                                                ),
+                                            )
+                                        }
+                                    >
+                                        {brick.usage}
+                                    </button>
+                                </p>
+                            ))}
+                        </div>
+                    </Tab.Panel>
+                ))}
+            </FilterTabs>
+        </>
+    );
+};
+
+export default AllTextBricks;
+
+export const getServerSideProps = async () => {
+    await dbConnect();
+    const result = await TextBrick!.find({}).exec();
```

Comment:
If you don't need hydration, you can speed up your queries quite a bit using [lean](https://mongoosejs.com/docs/tutorials/lean.html).

```suggestion
    const result = await TextBrick!.find().lean().exec();
```

Reply:
Resolved in [6eb8835](https://github.com/hpi-dhc/PharMe/pull/336/commits/6eb88359a5990d54e911a9d6b8f87e2c5a8db005)